.travis.yml

```diff
@@ -10,6 +10,8 @@ jobs:
     env: JOB_ARCHITECTURE=arm
   - os: linux
     env: JOB_ARCHITECTURE=aarch64
+  - os: linux
```

Comment:
Could we make this the first job so that we'll fail earlier?

Reply:
Done. I don't think Travis CI has the concept of job dependencies though, so I think it'll still run all of the jobs and will probably not send the failure email until they've all finished.